public/assets/css/ci-theme.css

```diff
@@ -2124,3 +2124,34 @@ html.dark-theme #discuss-icon-holder {
 html.dark-theme #discuss-icon {
     fill: rgba(221, 72, 20, 0.9);
 }
+
+.inner-page-text-box {
+    padding: 32px;
+}
+
+.cta-btn {
+    margin: 0;
+    float: none;
+    display: inline-block;
+}
+
+.cta-btn:not(:first-of-type) {
+    margin-left: 1rem;
+}
+
+ul li {
+    margin-top: 1rem;
+    line-height: 1.5em;
```

Comment:
How about this? @darthleslin @MGatner 

```suggestion
    line-height: 1em;
```
![Screenshot 2024-06-07 9 53 11](https://github.com/codeigniter4projects/website/assets/87955/f352114e-467c-46ba-959f-a6d342d39880)

Reply:
Looks great 👍.

Reply:
Excellent! Thank you for entertaining my nitpicks 🤗